backend/src/apiserver/client/argo.go

```diff
@@ -40,7 +40,7 @@ func (argoClient *ArgoClient) Workflow(namespace string) argoprojv1alpha1.Workfl
 
 func NewArgoClientOrFatal(initConnectionTimeout time.Duration, clientParams util.ClientParameters) *ArgoClient {
 	var argoProjClient argoprojv1alpha1.ArgoprojV1alpha1Interface
-	var operation = func() error {
+	operation := func() error {
```

Comment:
Same here, do we have to change the function definition here?

Reply:
This is the result of linting. It looks better stylistically and it actually means that we allocate memory for that function, which is later passed as an argument to another function.